src/pickup.cpp

```diff
@@ -285,8 +287,13 @@ void Pickup::pick_one_up( const point &pickup_target, item &newit, vehicle *veh,
                                                       g->u.remove_weapon(), 1 );
                             g->u.inv.assign_empty_invlet( newit, true ); // force getting an invlet.
```

Comment:
This will or at least might change the invlet. `use_invlet` is now outdated and could be wrong. Same below.